lib/index.js

```diff
@@ -228,3 +236,35 @@ FacebookPixel.prototype.completedOrder = function(track) {
 function formatRevenue(revenue) {
   return Number(revenue || 0).toFixed(2);
 }
+
+/**
+ * Get Traits Formatted Correctly for FB.
+ * 
+ * https://developers.facebook.com/docs/facebook-pixel/pixel-with-ads/conversion-tracking#advanced_match
+ * 
+ * @api private
+ */
+
+function formatTraits(analytics) {
+  var traits = analytics && analytics.user().traits();
+  if (!traits) return {};
```

Comment:
`analytics.user().traits()` always returns {} if there are no traits and empty objects are truthy.

I think you might want to explicitly check if this object is empty